packages/app/src/lib/auth/email.ts

```diff
@@ -19,7 +19,9 @@ export class EmailAuthClient implements AuthClient {
         const sent = new Date(sentAt);
         const message = async () => {
             const sentString = await formatDateFromNow(sent);
-            return html` <div>${$l(`Please enter the six digit verification code sent to {0}!`, email)}</div>
+            return html` <div class="break-words">
```

Comment:
As I was testing a really long email, I noticed it looked terrible (all in one line and not visible past 50 characters or so). This way it's visible and inside the box.